cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/batch/DynamicPartitionerWithAvroTest.java

```diff
@@ -114,6 +114,9 @@ public void apply() throws IOException {
           Map<PartitionKey, PartitionDetail> partitions = new HashMap<>();
           for (PartitionDetail partition : pfs.getPartitions(null)) {
             partitions.put(partition.getPartitionKey(), partition);
+            // check that the mapreduce wrote the output partition metadata to all the output partitions
```

Comment:
No, we don't have a test case for that specific example.
We do use two PFS as output in `DataCleansing` app, but don't write metadata to it.